Mathlib/RepresentationTheory/FDRep.lean

```diff
@@ -113,8 +112,11 @@ def of {V : Type u} [AddCommGroup V] [Module k V] [FiniteDimensional k V]
 instance : HasForget₂ (FDRep k G) (Rep k G) where
   forget₂ := (forget₂ (FGModuleCat k) (ModuleCat k)).mapAction (MonCat.of G)
 
-theorem forget₂_ρ (V : FDRep k G) : ((forget₂ (FDRep k G) (Rep k G)).obj V).ρ = V.ρ := by
-  ext g v; rfl
+/-- Lift `FdRep` to `Rep`. -/
+abbrev toRep (V : FDRep k G) : Rep k G := (forget₂ (FDRep k G) (Rep k G)).obj V
```

Comment:
I don't like this: as you immediately discover below, there are then too many ways to say the same thing, and naming becomes inconsistent.